app/assets/javascripts/listings/ListingService.js.coffee

```diff
@@ -322,12 +335,132 @@ ListingService = ($http, $localStorage, $modal, $q, $state) ->
     # by default will just return the id, unless it finds a matching slug
     return if mapping[slug] then mapping[slug] else id
 
-  Service.listingIs = (listing, name) ->
-    Service.LISTING_MAP[listing.Id] == name
+  Service.listingIs = (name) ->
+    Service.LISTING_MAP[Service.listing.Id] == name
 
   Service.listingIsAny = (listing, names) ->
```

Comment:
@chrisdolendo it's still used for some of the hardcoded things e.g. for Brighton, you even used it in your hotfix, haha... like for hiding DTHP it uses `ng-if="!listingIs('280 Brighton')"`, so this ListingService function is referred to in the ListingController and ShortFormApplicationController.